app/src/pages/inside/projectSettingsPageContainer/content/elements/fieldElement/fieldElement.scss

```diff
@@ -42,9 +42,16 @@
   line-height: 16px;
   font-family: $FONT-ROBOTO-REGULAR;
 
-  a {
+  span a {
```

Comment:
In my opinion it is better to add the new prop to FormattedDesc component.
It will allow us to do not rely on html structure

Reply:
Done.